botocore/exceptions.py

```diff
@@ -295,9 +295,11 @@ class ClientError(Exception):
         'operation: {error_message}')
 
     def __init__(self, error_response, operation_name):
+        code = error_response.get('Error', {}).get('Code', 'Unknown')
```

Comment:
It makes sense to handle the case where 'Code' can be None, but is there ever a case where `Error` will be None?  I don't think that' possible, this is a key we create in our response parser, not a value we get from the service.